### PR TITLE
fixes validation of min_length, max_length

### PR DESCRIPTION
**Pull request by jonasheinrich**
https://www.openproject.org/work_packages/14386
https://www.openproject.org/work_packages/14403

### CODE REVIEW

**Comment by hschink** [comment]
@jonasheinrich I miss specs.

**Comment by jonasheinrich** [comment]
@hschink There you go :)

**Comment by hschink** [comment]
@jonasheinrich Numbers smaller than zero are still accepted.

File: app/models/custom_value.rb [hunk 1]
```diff
@@ -94,7 +94,7 @@ def validate_type_of_value
   end
 
   def validate_length_of_value
-    if value.present?
+    if value.present? && custom_field.min_length.present? && custom_field.max_length.present?
```
Comment (hschink) [inline app/models/custom_value.rb:97]: There is no error message for the case that `min_length` or `max_lengt` are not set.

**Pull request merged**